### PR TITLE
add native arm64 builds for Windows

### PR DESCRIPTION
**Pull request by alanrenmsft**
For: https://github.com/microsoft/azuredatastudio/issues/1774

1. Enable Windows ARM64 builds
image

2. reorganized the download sections in readme file, the names are the same as in the VSCode download page.

**BEFORE**
image

**AFTER**
image

### CODE REVIEW

File: product.json [hunk 1]
```diff
@@ -15,11 +15,11 @@
 	"win32NameVersion": "Azure Data Studio",
 	"win32RegValueName": "azuredatastudio",
 	"win32AppId": "{{760C8610-682F-4E9B-9C9C-4E1120308DF2}",
-	"win32arm64AppId": "{{D1ACE434-89C5-48D1-88D3-E2991DF85479}",
+	"win32arm64AppId": "{{476E9913-B59B-492B-A75A-9C2866AFF86D}",
```
Comment (alanrenmsft) [inline product.json:18]: new guid for ADS, the old values are used by VSCode.

File: build/azure-pipelines/sql-product-build.yml [hunk 2]
```diff
@@ -106,17 +106,28 @@ stages:
...
+          VSCODE_ARCH: x64
         steps:
         - template: win32/sql-product-build-win32.yml
+        condition: and(succeeded(), eq(variables['VSCODE_BUILD_WIN32'], 'true'))
         timeoutInMinutes: 90
 
+      - job: Windows_ARM64
+        variables:
+          VSCODE_ARCH: arm64
+          RUN_TESTS: false # Do not run tests for arm64 build
```
Comment (Charles-Gagnon) [inline build/azure-pipelines/sql-product-build.yml:126]: Why not?

**Review by Charles-Gagnon — APPROVED** [review]

File: build/azure-pipelines/sql-product-build.yml (hunk 2 above)
Comment (alanrenmsft) [reply build/azure-pipelines/sql-product-build.yml:126]: already run tests in the x64 job, no additional coverage added.

Comment (Charles-Gagnon) [reply build/azure-pipelines/sql-product-build.yml:126]: I think we ideally should be doing tests on ALL architectures - who knows if some library has an issue on some specific arch. Does turning them on hurt/change anything?

At the very least could you come back in and update the comment to explain that? This comment is pretty useless right now - context without having to dig into PR histories is important for stuff like this

**Pull request merged**